evaluate/report/csv.go

```diff
@@ -121,3 +181,164 @@ func WriteEvaluationRecord(resultPath string, model model.Model, language langua
 
 	return nil
 }
+
+// loadEvaluationRecords reads and returns the evaluation records from the evaluation CSV file.
+func loadEvaluationRecords(resultPath string) (evaluationRecords EvaluationRecords, err error) {
+	evaluationFilePath := filepath.Join(resultPath, "evaluation.csv")
+	evaluationFile, err := os.Open(evaluationFilePath)
+	if err != nil {
+		return nil, pkgerrors.WithStack(err)
+	}
+	defer evaluationFile.Close()
+
+	reader := csv.NewReader(evaluationFile)
+
+	// Check if the evaluation CSV header is correct.
+	if header, err := reader.Read(); err != nil {
+		return nil, pkgerrors.Wrap(err, "file is empty")
+	} else if !reflect.DeepEqual(header, EvaluationHeader()) {
```

Comment:
This should be two strings, no?

Reply:
The `Read` function returns the header as a string slice, same type as `EvaluationHeader`

Reply:
maybe let's just do `strings.Join` and then compare the strings. Calling `reflect` feels a bit overkill

Reply:
Good call